pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-kb-ebsco-java</artifactId>
-  <version>0.1.0-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
just curious about major version change from 0.1.0 to 2.0.0. probably my mean 1.0.0?

Reply:
No -- good question @natali-zaitseva -- it is 2.0.0-SNAPSHOT as per suggestion/conversation with @funkymalc. The current plan is to make the version greater than the latest release version of mod-kb-ebsco. Some ongoing discussion around this here - https://issues.folio.org/browse/FOLIO-1727

Reply:
Thank you for explanation @carolegodfrey!